addons/mail/static/src/core/common/thread.js

```diff
@@ -178,6 +179,11 @@ export class Thread extends Component {
                 toRaw(nextProps.thread).fetchNewMessages();
             }
         });
+        onRendered(() => {
+            if (this.env.inDiscussApp) {
+                this.env.config.setDisplayName(this.props.thread.displayName);
```

Comment:
Does it have an effect on public page or not? And if so, do we want it?
And if not, it should be in web bundle instead of common.